### PR TITLE
print an intro message, this follows up on #1894

### PR DESCRIPTION
**Pull request by robertoostenveld**

### CODE REVIEW

**Comment by robertoostenveld** [comment]
```
>> clear all; restoredefaultpath
>> ft_defaults
-------------------------------------------------------------------------------------------
FieldTrip is developed by members and collaborators of the Donders Institute for Brain,
Cognition and Behaviour at the Radboud University, Nijmegen, the Netherlands.

Please cite the FieldTrip reference paper when you have used FieldTrip in your study.
Robert Oostenveld, Pascal Fries, Eric Maris, and Jan-Mathijs Schoffelen. FieldTrip: Open
Source Software for Advanced Analysis of MEG, EEG, and Invasive Electrophysiological Data.
Computational Intelligence and Neuroscience, vol. 2011, Article ID 156869, 9 pages, 2011.
doi:10.1155/2011/156869.
-------------------------------------------------------------------------------------------
Warning: FieldTrip is not yet on your MATLAB path, adding /Users/roboos/matlab/fieldtrip 
```

@schoffelen what do you think of this? We could also add the mention of Donders and RU to the website home page. It was there before, but apparently was dropped at a certain time,

File: ft_defaults.m [hunk 1]
```diff
@@ -147,6 +147,20 @@
   prevcleanup = {};
 end
 
+% show a welcome message
+fprintf([
+  '-------------------------------------------------------------------------------------------' ...
+  '\nFieldTrip is developed by members and collaborators of the Donders Institute for Brain,' ...
+  '\nCognition and Behaviour at the Radboud University, Nijmegen, the Netherlands.' ...
```
Comment (schoffelen) [inline ft_defaults.m:154]: I think that 'the' can be omitted

**Review by schoffelen — CHANGES_REQUESTED** [review]

**Comment by schoffelen** [comment]
Looks good. Perhaps it's indeed diplomatically and strategically wise to also mention the DI + RU on the website.

File: ft_defaults.m (hunk 1 above)
Comment (robertoostenveld) [reply ft_defaults.m:154]: you mean the 2nd 'the' for Radboud?

Comment (schoffelen) [reply ft_defaults.m:154]: yes, I think Radboud University is without 'the'

**Pull request merged**